edx-platform/pearson-vue-theme/lms/templates/index.html

```diff
@@ -11,17 +11,33 @@
     <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
   <div class="signin-container">
     <div class="toggle-form">
-      <h1>${_("Welcome!")}</h1>
-      <br>
-      <span class="text">${_("Sign in or create an account to access your courseware.")}</span>
+      <span class="text">${_("Welcome to CertPREP courseware!")}</span>
+      <span class="text-bold">${_("Everything in one spot for a seamless experience.")}</span>
+      <span class="text">${_("All text, videos, quizzes, practice tests and labs are integrated within the courseware with one consistent interface and log-in for instructors and students.")}</span>
     </div>
     <div class="login-providers">
-      <a href="/login">
-        <button class="button-vue">
-          <span aria-hidden="true">${_("Continue")}</span>
-          <span class="icon fa fa-arrow-right"></span>
-        </button>
-      </a>
+      <div class="button-container">
+        <div>
+          Sign in to access you courseware
+        </div>
+        <a href="/login">
+          <button class="button-vue">
+            <span aria-hidden="true">${_("Continue")}</span>
+            <span class="icon fa fa-arrow-right"></span>
+          </button>
+        </a>
```

Comment:
```suggestion
        <a class="btn btn-primary btn-small" href="/login" role="button">
            <span aria-hidden="true">${_("Continue")}</span>
            <span class="icon fa fa-arrow-right"></span>
        </a>
```
What do you think about using the bootstrap classes for buttons instead of .button-vue?

Reply:
Thanks I was wondering if there was any bootstrap class for this